python/hsfs/feature_store.py

```diff
@@ -161,19 +161,43 @@ def get_feature_groups(self, name: str):
         )
 
     def get_on_demand_feature_group(self, name: str, version: int = None):
-        """Get a on-demand feature group entity from the feature store.
+        """Get a external feature group entity from the feature store.
 
-        Getting a on-demand feature group from the Feature Store means getting its
+        !!! warning "Deprecated"
```

Comment:
You added the deprecation note, but removed the method from the auto_doc.py so it won't show up anywhere.